### ISSUE TITLE
Lucky Blocks spawning in water only at y=62

### ISSUE THREAD

**Issue by GitNuck** [TheBusyBiscuit/luckyblocks-sf#41]
## Description;
Using the latest Slimefun dev versions on Purpur 1.16.5 1171, Lucky Blocks will spawn only in water 1 block deep at Y=62
This has been happening for quite a while over several versions, users thought it was coded to be like that.

## Steps to reproduce the Issue;
Load the latest dev of Slimefun 4 and Lucky blocks and run.  Changing the chance multiplier has no effect.

## Expected behavior;
Looking at the luckyblocks-sf-dev 34 jar file in world.generator.class, a lucky block should be randomly generated in a chunk's highest "Y" position on a solid block.

## Server Log / Error Report;
No error files generated.

## Environment;
![2021-08-16_01 40 30](https://user-images.githubusercontent.com/88690764/129528651-7615a030-7549-485f-b784-fccfd90e0cd8.png)

**Comment by GitNuck** [comment]
I noticed something with luckyblock Y level generation the other day after updating.  I teleported out to a new chunk and set my Y co-ordinate to 110.  I spawned into a mountain biome and at Y=74 in one block deep water in a tiny pond was a luckyblock.  The plugin seems to be instantly calculating the Y value at the feet of the player, like how a chunk starts generating in the game.  I don't know why it's always in water 1 block deep though, perhaps water is generated first in the chunk and the plugin sees nothing but water to place blocks.
